app/controllers/automation_manager_controller.rb

```diff
@@ -8,6 +8,7 @@ class AutomationManagerController < ApplicationController
   include Mixins::GenericSessionMixin
   include Mixins::ManagerControllerMixin
   include Mixins::ExplorerPresenterMixin
+  include EmsCommon
```

Comment:
This one frightens me. `EmsCommon` is a hydra. I am more for splitting it and cleaning up rather then adding it to more places.